Content.Server/Chemistry/ReagentEffects/PlantMetabolism/PlantSedin.cs

```diff
@@ -0,0 +1,49 @@
+using Content.Server.Botany.Components;
+using Content.Server.Botany.Systems;
+using Content.Shared.Chemistry.Reagent;
+using JetBrains.Annotations;
+using Content.Server.Popups;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Random;
+
+namespace Content.Server.Chemistry.ReagentEffects.PlantMetabolism
+{
```

Comment:
use `namespace ...;` not `namespace ... {`

Reply:
namespace ... { was used in Robust Harvest, which is what I based the code for this off of

Reply:
doesn't matter, the codebase standard is `namespace ...;`